status-im/status-react
# [FIX #3172] Wipe db when logging out

fix #3172 

To test:

check that you can:
- create an account, login logout, can't go back from profile screen to account, 
- login in other account
- recover account
- switch network
